src/server/req_rescq.c

```diff
@@ -222,6 +222,199 @@ cnvrt_timer_init()
  */
 
 
+/**
+ * @brief
+ * 		remove_node_from_resv - procedure removes node from reservation
+ *		the node is removed from RESV_ATR_resv_nodes and assigned
+ *		resources are accounted back to loaner's pool and finally the
+ *		reservation is removed from the node
+ *
+ * @parm[in,out]	presv	-	reservation structure
+ * @parm[in,out]	pnode	-	pointer to node
+ *
+ */
+void
+remove_node_from_resv(resc_resv *presv, struct pbsnode *pnode) {
```

Comment:
'{' should be on the first column of the next line. Similar change is also needed in the newly added functions.

Reply:
Done.